dimmer.el

```diff
@@ -264,6 +267,13 @@ uses a buffer name that fits this pattern."
     (add-to-list
      'dimmer-prevent-dimming-predicates #'which-key--popup-showing-p)))
 
+;;;###autoload
+(defun dimmer-configure-magit ()
+  "Convenience settings for magit users."
+  (with-no-warnings
```

Comment:
Technically we don't need a `with-no-warnings` here. There are cases where we need to refer to predicate functions in other packages and the bytecompiler will complain that it doesn't recognize the symbol. But here the regexp doesn't depend on symbols in the package.